data/data/gcp/network/lb-private.tf

```diff
@@ -19,6 +19,9 @@ resource "google_compute_health_check" "api_internal" {
     port         = 6443
     request_path = "/readyz"
   }
+  log_config {
```

Comment:
[e2e-gc][1]:

```
level=error msg=Error: Unsupported block type
level=error
level=error msg=  on ../tmp/openshift-install-1114*2364/network/lb-private.tf line 22, in resource "google_compute_health_check" "api_internal":
level=error msg=  22:   log_config {
level=error
level=error msg=Blocks of type "log_config" are not expected here.
level=error
level=error msg=Failed to read tfstate: open /tmp/openshift-install-1114*2364/terraform.tfstate: no such file or directory
level=fatal msg=failed to fetch Cluster: failed to generate asset "Cluster": failed to create cluster: failed to apply Terraform: failed to complete the change
```

Maybe a Terraform version thing?  Because what you have here matches [the current provider docs][2], as far as I can tell.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/4998/pull-ci-openshift-installer-master-e2e-gcp/1404536721646817280
[2]: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_health_check#log_config

Reply:
Yeah, seems likely.  I'll try to bump it later, since it's not that new.